Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -206,10 +206,12 @@ final class RewardsCollectionViewController: UICollectionViewController {
     let adjustedContentInset = collectionView.adjustedContentInset
 
     let topBottomSectionInsets = sectionInsets.top + sectionInsets.bottom
+    let leftRightSectionInsets = sectionInsets.left + sectionInsets.right
     let topBottomContentInsets = adjustedContentInset.top + adjustedContentInset.bottom
 
     let itemHeight = collectionView.frame.height - topBottomSectionInsets - topBottomContentInsets
-    let itemWidth = CheckoutConstants.RewardCard.Layout.width
+    let widthByBounds = self.view.bounds.size.width - leftRightSectionInsets
```

Comment:
We could probably simply use `collectionView.frame.width` to not having to call `self` here ... in case you'd still prefer to use `self` it could be shortened to use the getter on `CGRect` like so `self.view.bounds.width`.